posthog/models/event.py

```diff
@@ -244,7 +246,9 @@ def create(self, site_url: Optional[str] = None, *args: Any, **kwargs: Any):
                         should_post_webhook = True
                 Action.events.through.objects.bulk_create(relations, ignore_conflicts=True)
                 team = kwargs.get("team", event.team)
-                if should_post_webhook and team and team.slack_incoming_webhook:
+                if (
+                    should_post_webhook and team and team.slack_incoming_webhook and not is_ee_enabled()
```

Comment:
only trigger the hook on PG side if EE isn't enabled. EE will handle it separately